src/libraries/Microsoft.Extensions.Logging/gen/Resources/Strings.resx

```diff
@@ -0,0 +1,213 @@
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="InvalidLoggingMethodNameMessage" xml:space="preserve">
+    <value>Logging method names cannot start with _</value>
+  </data>
+  <data name="InvalidLoggingMethodParameterNameMessage" xml:space="preserve">
+    <value>Logging method parameter names cannot start with _</value>
+  </data>
+  <data name="LoggingMethodInNestedTypeMessage" xml:space="preserve">
+    <value>Logging class cannot be in nested types</value>
+  </data>
+  <data name="MissingRequiredTypeTitle" xml:space="preserve">
+    <value>Could not find a required type definition</value>
+  </data>
+  <data name="MissingRequiredTypeMessage" xml:space="preserve">
+    <value>Could not find definition for type {0}</value>
+  </data>
+  <data name="ShouldntReuseEventIdsTitle" xml:space="preserve">
+    <value>Multiple logging methods cannot use the same event id within a class</value>
+  </data>
+  <data name="ShouldntReuseEventIdsMessage" xml:space="preserve">
+    <value>Multiple logging methods are using event id {0} in class {1}</value>
+  </data>
+  <data name="LoggingMethodMustReturnVoidMessage" xml:space="preserve">
+    <value>Logging methods must return void</value>
+  </data>
+  <data name="MissingLoggerArgumentMessage" xml:space="preserve">
+    <value>One of the arguments to a logging method must implement the Microsoft.Extensions.Logging.ILogger interface</value>
```

Comment:
You should probably make strings that reference types locked strings. Here is an example: https://github.com/dotnet/runtime/blob/a551a99ed3f36284816123ae334eb8c58eeee184/src/libraries/System.Private.CoreLib/src/Resources/Strings.resx#L483